task.sh

```diff
@@ -6,44 +6,43 @@
 #   dir2
 #   dir3
 #       dir4
-mkdir -p /task/task
-mkdir dir1 dir2 dir3
-mkdir ./dir3/dir4
+# как же долго до меня доходит!!!
+mkdir -p task/dir1 task/dir2 task/dir3/dir4
 # изменяем текущую директорию на task
-cd /task/task
+cd ../task
```

Comment:
Так если мы создаем папку `task` рядом со скриптом, то зачем мы поднимаемся на уровень выше?
```suggestion
cd task
```

Reply:
соре, читаю одним местом
не видел коммент в практикуме
некст запушу сюда